src/components/shared/EditText/index.tsx

```diff
@@ -160,143 +193,281 @@ function EditText(props: Props): ReactElement {
     numberOfLines,
   } = props;
 
-  if (isRow) {
-    return (
-      <StyledRowContainer style={style}>
-        <StyledRowContent
-          style={[
-            { borderColor: borderColor },
-            focused
-              ? { borderColor: focusColor }
-              : errorText
-                ? { borderColor: errorColor }
-                : null,
-            inputContainerType === 'underlined' || !inputContainerType
-              ? [{ borderBottomWidth: 0.6 }]
-              : { borderWidth: 0.6, borderRadius: inputContainerRadius },
-          ]}
-        >
-          {label ? (
-            <StyledRowLabel
-              style={[
-                labelTextStyle,
-                errorText
+  switch (type) {
+    case EditTextInputType.DEFAULT:
```

Comment:
Try to make this case as `default:`. Not putting `default` condition may result in returning `undefined`.

Reply:
Thank you!!😆